src/Encamina.Enmarcha.SemanticKernel/MemoryManager.cs

```diff
@@ -20,29 +20,30 @@ public class MemoryManager : IMemoryManager
 {
     private const string ChunkSize = @"chunkSize";
 
-    private readonly Kernel kernel;
-
     private readonly ILogger logger;
 
+    private readonly ITextEmbeddingGenerationService textEmbeddingGenerationService;
+
     /// <summary>
     /// Initializes a new instance of the <see cref="MemoryManager"/> class.
     /// </summary>
     /// <param name="kernel">
     /// A valid instance of <see cref="Kernel"/>, used to get the configured text embeddings generation service (<see cref="ITextEmbeddingGenerationService"/>) required by this manager.
     /// </param>
     /// <param name="memoryStore">A valid instance of a <see cref="IMemoryStore"/> to manage.</param>
```

Comment:
Comments on the new parameters are missing

Reply:
True. These quality details are going to be address completely once the final version is push for review.

Reply:
Ok